.github/workflows/eval_test.yaml

```
@@ -0,0 +1,60 @@
+name: Evaluation Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Weekdays nightly at 07:00 UTC = 23:00 PST / 00:00 PDT.
+    - cron: "0 7 * * 1-5"
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  test_perplexity:
+    name: "Evaluation Tests - perplexity"
+    strategy:
+      matrix:
+        version: [3.11]
+        os: [llama-mi300]
+      fail-fast: false
+    runs-on: ${{matrix.os}}
```

llama-mi300 is a runner label, not an operating system. The fields under `matrix` can have any name you want, so I'd change to something like `runs-on` to make the usage obvious:

```suggestion
    strategy:
      matrix:
        version: [3.11]
        runs-on: [llama-mi300]
      fail-fast: false
    runs-on: ${{matrix.runs-on}}
```